aten/src/ATen/native/quantized/cudnn/utils.h

```diff
@@ -192,6 +192,17 @@ struct TORCH_API PackedConvWeightCudnn : public ConvPackedParamsBase<kSpatialDim
 namespace cudnn_utils {
 namespace {
 
+// TODO: we can remove this function when cuDNN enables pass by value support for
+// pointwise multiplication operations. the only reason why we need this right now is
+// we use broadcasting scalar multiplication in conv, linear, and add ops, and cuDNN requires
+// the scalar to be a scalar tensor with the same number of dimensions (num_dim) as the tensor we're multiplying to
+at::Tensor get_requant_multiplier_tensor(double requant_multiplier, uint8_t num_dim) {
```

Comment:
nit: maybe align the naming, looks like we should be using getReauntMultiplerTensor from the other names in this file